entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/extension/KotlinTypeExt.kt

```diff
@@ -1,12 +1,20 @@
 package godot.entrygenerator.extension
 
 import org.jetbrains.kotlin.types.KotlinType
+import org.jetbrains.kotlin.types.typeUtil.supertypes
 
 
 fun KotlinType.isCoreType(): Boolean {
     return coreTypes.contains(this.toString())
 }
 
+fun KotlinType.isResource(): Boolean {
+    return this.toString() == "GodotResource"
```

Comment:
are we able to check the fully qualified name? Also shouldn't this be `Resource`?

Reply:
with this yes `kotlinType.getJetTypeFqName(false)` but maybe there is a better solution?

Reply:
currently it's named `GodotResource` @piiertho whats the final name?

Reply:
It is named Resource :)

Reply:
I renamed it :-)